roles/idrac_server_powerstate/meta/argument_specs.yml

```diff
@@ -67,3 +71,12 @@ argument_specs:
             "PowerCycle",
             "PushPowerButton"
           ]
+      oem_reset_type:
+        description:
+          - This parameter initiates a complete Alternate Current (AC) power cycle of the server which is equivalent to disconnecting power
+            cables using OEM API.
+          - I(oem_reset_type) is mutually exclusive with I(reset_type).
+          - If the value of 'final_power_state' is not provided, the default value is 'Off'.
+        required: false
```

Comment:
can't we add suboptions for reset_type and final_power_state? Could you tell me how you ensure the user will pass the same key(reset_type, final_powers_state)?

Reply:
I think it's fine because Oem will differ.